### PR TITLE
A couple of current muses

### PR DESCRIPTION
**Pull request by jcorbin**

### CODE REVIEW

File: bandnames.txt [hunk 1]
```diff
@@ -106,6 +107,7 @@ The Amazing Mumfords
 The Baby Boom
 The Eigenotes
 The Immortal Coils
+The Irregular Polyhedrons (via jcorbin)
```
Comment (kriskowal) [inline bandnames.txt]: Or Polyhedra

Polly Hedra!

Comment (kriskowal) [reply bandnames.txt]: Entomologist D&D campaign: Rollypolyhedra

**Comment by kriskowal** [comment]
To win at CI, preserve sort order.

**Comment by kriskowal** [comment]
The CI job might need to be altered to be consistent with `sort -f` instead of merely `sort`.

**Comment by jcorbin** [comment]
I mean, the prior wasn't sorted according to my sort and eye... so idk :-/

**Comment by jcorbin** [comment]
also I don't know how github actions work, but it seems to be checking out and testing against the previous commit after I push a new one (force or otherwise)...

**Comment by jcorbin** [comment]
I have smote the CI foe

**Pull request merged**